src/Features/Core/Portable/BraceCompletion/IBraceCompletionService.cs

```diff
@@ -64,9 +64,9 @@ internal struct BraceCompletionResult
         /// The caret location in the new text created by applying all <see cref="TextChanges"/>
         /// to the input text.
         /// </summary>
-        public int CaretLocation { get; }
+        public LinePosition CaretLocation { get; }
```

Comment:
might be worth pointing out that this can be 'virtual' (or however you want to explain that it might refer to a desired column that doesn't point at actual whitespace.